docs/README.md

```diff
@@ -1,47 +1,43 @@
-# Overview
-**InsertionsClient** updates the versions of NuGet packages in _default.config_ with the corresponding versions specified in _manifest.json_ assets. It also updates the values of properties defined in .props files.
-
-## How does **InsertionsClient** work?
-1. Loads into memory the contents of both _default.config_ and _manifest.json_ as well as all the _.packageconfig_ files listed in the _default.config_
-1. Searches _default.config_ and _.packageconfig_ files for corresponding NuGet packages for each _manifest.json_ asset
-1. For every match, the NuGet version in the config file is updated with that of the corresponding _manifest.json_ asset
-1. The updated _default.config_ and _.packageconfig_'s are saved on disk
+# InsertionsClient
+The **InsertionsClient**, .NET core commnand line executable, updates _NuGet_ packages in files relevant to SDK insertions, such as 
+_default.config_ and .props, with corresponding _manifest.json_ versions.
+
+**Table of Contents**
+- [How it works](#howitworks)
+-  &nbsp;&nbsp;[Input](#input)

```

Comment:
minor: This nested list looks strange, it has all the points on the top level. Why not like the following?
```
**Table of Contents**
- [How it works](#howitworks)
  - [Input](#input)
  - [Log](#log)
  - [Use](#use)
  - [Output](#output)
- [Code Base](#codebase)
  - [Microsoft.DotNet.InsertionsClient.Core](#corelib)
    - [Examples](#corelibexamples)
  - [Microsoft.DotNet.InsertionsClient.Console](#console)
    - [Examples](#consoleexample)
```

Reply:
Updated as suggested.